### ISSUE TITLE
Images not displayed when attached in after hook. Works in afterstep hook

### ISSUE THREAD

**Issue by sebinjoseph1994** [grasshopper7/cucumber-pdf-plugin-report#1]
Thanks much for the plugin. Just wanted to know whether I can see the screenshots attached using cucumber.attach in the report in the PDF as well. Could you please guide me on this, if this is supported?

**Comment by grasshopper7** [comment]
Please refer to this - https://github.com/grasshopper7/cucumber-pdf-plugin-report/blob/7eb8cf196aedbd830c4fa731d585f568d6c75696/cucumber-pdf-plugin-report/src/test/java/stepdefs/Stepdefs.java#L150. 

U should get something like this. A thumbnail as well as a zoomed image.
[report.pdf](https://github.com/grasshopper7/cucumber-pdf-plugin-report/files/6405048/report.pdf)

**Comment by sebinjoseph1994** [comment]
@grasshopper7 Thanks for the reply. I'm using the same step but inside @After hook. I'm taking screenshots when there is a failure in the test from @After hook of cucumber. Unfortunately, I'm not able to see the screenshot in the report.
```
 @After
    public void after(Scenario scenario) {
        logger.info("Running after hook");
        if(scenario.isFailed()) {
            byte[] ss = (TakesScreenshot) driver).getScreenshotAs(OutputType.BYTES);
            scenario.attach(ss, "image/png", scenario.getName());
        }
    }
```

Thanks in advance.

**Comment by sebinjoseph1994** [comment]
@grasshopper7  This is the report file being generated for me.
[report.pdf](https://github.com/grasshopper7/cucumber-pdf-plugin-report/files/6405201/report.pdf)

I could see the Embedded folder is being created in my project by the plugin, but the image is not available in the report.

**Comment by grasshopper7** [comment]
Is the image file generated in the embedded folder?

**Comment by sebinjoseph1994** [comment]
Yes it is generated

**Comment by sebinjoseph1994** [comment]
On using it on @AfterStep it is working as expected, but not with @After. I can replace the failure check to @AfterStep from @After. Thank you so much for the quick support :)

**Comment by grasshopper7** [comment]
This is a huge bug. Are u using the latest version?

**Comment by sebinjoseph1994** [comment]
1.4.1

**Comment by grasshopper7** [comment]
Then it is a problem. I will test and fix it. Thanks.BTW are u seeing the thumbnail and expanded image?

**Comment by sebinjoseph1994** [comment]
Sure, thank you.

**Comment by grasshopper7** [comment]
Have fixed it. Will release a new version to Maven central. Should be available soon.

**Comment by sebinjoseph1994** [comment]
The Before was getting executed actually instead of After, right? I didn't have Before hook in my code earlier, but when added that noticed this is the issue :)

**Comment by grasshopper7** [comment]
Something like that. There was a copy-paste error and there was no test for after hook. Fixed in version 1.4.2 which is available in maven central now.
